docs/using/tutorial/movingvolume.rst

```diff
@@ -0,0 +1,231 @@
+.. _tutorial-moving-a-data-volume:
+
+==============================
+Tutorial: Moving a Data Volume
+==============================
+
+In this tutorial (30 minutes)
+-----------------------------
+
+You will use Flocker to migrate a Docker container with its data volume from one host to another.
+The container you move will be part of a two-container application, the other container will not move and the two will remain connected even when they are on different hosts.
+
+To begin the tutorial you will first install the Flocker client on your local machine, then install Flocker onto two hosts.
+You will then be ready to use Flocker to migrate a Docker container with a volume attached from one host to the other.
+
+.. note:: This tutorial takes roughly 30 minutes, but because there are a few things to download, times might vary depending on the speed of your connection.
```

Comment:
If you put a time guide in this, please at least try it with the longest option (probably setting up either AWS or Rackspace) even with a good internet connection and see if you can do it in that time. I don't think that I could.